config/stadtwerk_tauberfranken.json

```diff
@@ -3,36 +3,34 @@
         "pricing_plans": [
             {
                 "currency": "EUR",
-                "description": "Tarif: 0,20€/km zzgl. 3€/h bzw. 30€/Tag bzw. 50€ am Wochenende (Freitag 17:00 Uhr bis Sonntag 23:59 Uhr). Einmalige Registrierung für 4,90€ erforderlich. Alle Angaben ohne Gewähr.",
+                "description": "Für jede Buchung setzen sich die Nutzungskosten zusammen aus Zeittarif + Kilometertarif.",
```

Comment:
You don't want to provide the initial registration fees and warranty disclaimer any longer?

Reply:
I added specifiy descriptions for each pricing plan.